### PR TITLE
(stoa0040)_many_many_files

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Continuing work on Augustine _cts_.xml, headers and bibliographic info.

### CODE REVIEW

File: data/stoa0040/stoa014a/stoa0040.stoa014a.opp-lat1.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
       </publicationStmt>
       
       <sourceDesc>
-        <listBibl xml:lang="lat">
+        <listBibl>
```
[the remark this answers is not in the document]
Comment (lcerrato) [reply data/stoa0040/stoa014a/stoa0040.stoa014a.opp-lat1.xml:45]: @AlisonBabeu 
Yes, I don't think it's warranted unless there are multiple sources. It should be fine to remove.

**Comment by lcerrato** [comment]
I thought we were going to limit the size of the PRs? ;)

**Review by lcerrato — APPROVED** [review]
I think we should address the `<listBibl>` tags in a separate pass, perhaps throughout the repo.

**Comment by AlisonBabeu** [comment]
Sorry we were going to limit the size of the PRs I just got carried away with my zeal to finish the headers of Augustine which alas won't happen as soon as I'd like.  And since the `listBibl` tag is in all of the CSEL files I believe it will definitely be something to script at the end.

**Pull request merged**